pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1683,18 +1675,18 @@ func TestPriorityQueue_MoveAllToActiveOrBackoffQueue(t *testing.T) {
 	}
 	expectInFlightPods(t, q, medPriorityPodInfo.Pod.UID)
 
-	q.schedulingCycle++
-	q.activeQ.AddOrUpdate(q.newQueuedPodInfo(unschedulablePodInfo.Pod))
+	q.activeQ.schedulingCycleInc()
```

Comment:
Ideally we should remove this method, and call pop to cause the scheduling cycle to increase.

Check if it's doable. But if it increases the diff too much, we can leave it for a follow up.

Reply:
It seems it was redundant in the tests, because scheduling cycle was still incremented in the `Pop` above. Removed the method